web/src/scss/_base.scss

```
@@ -0,0 +1,124 @@
+* {
+    /* Remove tap feedback on mobile */
+    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+    -moz-tap-highlight-color: rgba(0, 0, 0, 0);
+    -webkit-font-smoothing: antialiased;
+}
+
+a {
+    text-decoration: none;
+    color: #4178AC;
+
+    &:hover {
+	    cursor: pointer;
+	    text-decoration: underline;
+	}
+}
+
+html, 
+body {
+    margin: 0 !important;
+    padding: 0 !important;
+    font-family: $primary-font;
+    transition: background-color 800ms;
+}
+
+body {
+    height: 99.9%; /* Due to a bug (box sizing?), we can't set 100% (the map gets undraggable on Safari iPhone 5S) */
+}
+
+.all-screens-hidden {
+    display: none !important;
+}
+
+.large-screen-hidden {
+    @include respond-to('medium-up') {
+        display: none;
+    }
+}
+
+.small-screen-hidden {
+    @include respond-to('small') {
+        display: none;
```

!important flag is needed, else the left-panel contents won't properly switch from country list to detail.
```suggestion
        display: none !important;
```